desk/app/groups.hoon

```diff
@@ -183,6 +183,10 @@
       %group-cancel
     =+  !<(=flag:g vase)
     ga-abet:ga-cancel:(ga-abed:gang-core flag)
+  ::
+      %group-clone
+    =+  !<(=flag:g vase)
```

Comment:
This is a bit misleading because it uses a mark that actually exists, but with a different type. Assuming this is for local-only use (in which case we might also want a `?>  =(our src):bowl`), probably fine to rename to `%group-do-clone` or w/e.